### PR TITLE
feat: Navbar, Filters Sidebar responsive drawers, search feature

### PR DESCRIPTION
**Pull request by bharati-21**
This PR contains the following features:
- Created mobile view sidebars for navbar links, navbar search, and filters.
- Added the search feature

### CODE REVIEW

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -1,32 +1,122 @@
...
+            navigate('/products');
+        }
+    }
+    
+    const handleChangeSearchText = event => {
+        filterDispatch({ filterType: 'SET_SEARCH_TEXT', filterPayload: event.target.value });
+    }
+
+    const totalCartItems = isAuth ? cartItems?.length ? getCartItemsTotal(cartItems, "TOTAL_ITEMS") : 0 : 0;
+    const totalWishListItems = isAuth ? wishListItems?.length ? wishListItems.length : 0 : 0;
```
Comment (rahulyadav139) [inline src/components/Navbar/Navbar.jsx:59]: This syntax is new to me. Chaining in template literal. Great work.

File: src/components/FilterOptions/ClearFilters.jsx [hunk 2]
```diff
@@ -1,21 +1,34 @@
...
+					onClick={handleClearFilters}
+				>
+					Clear Filters
+				</button>
+				{
+                    showCloseIcon ? (
+					    <button className="btn btn-icon" onClick={() => handleChangeShowFilterDrawer(false)}>
+						    <CloseIcon fontSize="large" />
+					    </button>
+				    ) :
```
Comment (rahulyadav139) [inline src/components/FilterOptions/ClearFilters.jsx:26]: I didn't find anything special for correction. Clean code great naming convention. Import export is great, you are using barrel imports/exports. Great work.

**Review by rahulyadav139 — APPROVED** [review]

**Review by sonishreyas — APPROVED** [review]
code looks good and easy to understand

File: src/components/Navbar/Navbar.jsx (hunk 1 above)
Comment (bharati-21) [reply src/components/Navbar/Navbar.jsx:59]: Thank you

File: src/components/FilterOptions/ClearFilters.jsx (hunk 2 above)
Comment (bharati-21) [reply src/components/FilterOptions/ClearFilters.jsx:26]: Thank you for your review!

**Comment by bharati-21** [comment]
> code looks good and easy to understand

Thank you for your review!

**Pull request merged**